packages/epo-react-lib/src/atomic/Toast/Toast.tsx

```diff
@@ -0,0 +1,22 @@
+import { FunctionComponent, ReactNode } from "react";
+
+interface ToastProps {
+  children: ReactNode;
+  className?: string;
+  /** list of ID's that contributed to this status message eg. if clicking on \<svg id="mySvg"/> created a message, pass ['mySvg'] */
+  forIds?: string[];
+}
+
+const Toast: FunctionComponent<ToastProps> = ({
```

Comment:
this is an unstyled `Toast` component design to create a good basis for screenreader-ready status announcements. It is based on this: https://web.dev/building-a-toast-component/